tests/pdk/finfet_pdk/test_placement_grid.py

```diff
@@ -51,3 +52,19 @@ def test_scalings():
             "ored_terms": [{"offsets": [0], "scalings": [1, -1]}]
         }]
         assert primitive['metadata']['constraints'] == golden
+
+
+def test_hierarchy():
+    os.environ['PLACE_ON_GRID'] = 't'
```

Comment:
@854768750 This is the environment variable you need to set to enable the behavior you want with schematic2layout.py

Reply:
@stevenmburns Thanks. Let me try this.

Reply:
@stevenmburns The placement looks correct while there are some assertions. One possible reason is that unit in PnR is different from gds.json

value_in_json/ScaleFactor*2=value_in_PnR, here ScaleFactor=10 in layers.json

Reply:
@854768750 Do you want me to change the units to in PnR form?

Reply:
I have transformed the unit into PnR form in placer.
When checking the placement solution, it should be the same unit as the constraint.